src/ovirtapi/transport.js

```diff
@@ -57,27 +57,32 @@ type GetRequestType = { url: string, custHeaders?: Object}
 type InputRequestType = { url: string, input: string, contentType?: string }
 type DeleteRequestType = { url: string, custHeaders?: Object }
 
+let getCounter = 0
+const logHeaders = (headers) => JSON.stringify({ ...headers, 'Authorization': '*****' })
+
 function httpGet ({ url, custHeaders = {} }: GetRequestType): Promise<Object> {
-  logger.log(`_httpGet start: url="${url}"`)
+  const myCounter = getCounter++
   const requestId = notifyStart('GET', url)
-  const headers = Object.assign({
+  const headers = {
+    'Accept': 'application/json',
     'Authorization': `Bearer ${_getLoginToken()}`,
     'Accept-Language': AppConfiguration.queryParams.locale, // can be: undefined, empty or string
     'Filter': Selectors.getFilter(),
-    'Accept': 'application/json',
-  }, custHeaders)
-  logger.log(`_httpGet: url="${url}", headers="${JSON.stringify(headers)}"`)
+    ...custHeaders,
+  }
 
+  logger.log(`http GET[${myCounter}] -> url: "${url}", headers: ${logHeaders(headers)}`)
```

Comment:
As I see in other files `logger` was rewritten to `console` is there any reason to leave it here?